semanticcpg/src/main/scala/io/shiftleft/semanticcpg/codedumper/CodeDumper.scala

```diff
@@ -74,6 +76,14 @@ object CodeDumper {
                       line
                   }
                   .mkString("\n")
+              } else if (m.file.exists(_.content != PropertyDefaults.Content)) {
+                sliceCode(
```

Comment:
there's a `m.content` step already that should help here

Reply:
ah no, this code isn't dumping the method but (potentially) parts of it. Won't help much then.

Reply:
Thanks! That simplifies things. Making the related change now

Reply:
> ah no, this code isn't dumping the method but (potentially) parts of it. Won't help much then.

Ok, let's see if the tests pass, but I've used `m.content` and then split the lines so that the comment arrow can still work